znai/src/main/java/com/twosigma/znai/website/WebSite.java

```diff
@@ -472,7 +483,7 @@ private HtmlPageAndPageProps generatePage(TocItem tocItem, Page page) {
             Path pagePath = tocItem.isIndex() ? Paths.get("index.html") :
                     Paths.get(tocItem.getDirName()).resolve(tocItem.getFileNameWithoutExtension()).resolve("index.html");
 
-            Path originalPathForLogging = cfg.docRootPath.toAbsolutePath().relativize(
+            Path originalPathForLogging = cfg.docRootPath.relativize(
                     markupParsingConfiguration.fullPath(componentsRegistry, cfg.docRootPath, tocItem).toAbsolutePath());
```

Comment:
Don't you need to pass non AbsolutePath() for `cfg.docRootPath` to `markupParsingConfiguration.fullPath `?

Reply:
Could you remind me why? I was able to build blog engine with this changes as well.